libbeat/metric/system/process/process.go

```diff
@@ -483,7 +483,7 @@ func (procStats *Stats) getSingleProcess(pid int, newProcs ProcsMap) *Process {
 
 	err = process.getDetails(procStats.isWhitelistedEnvVar)
 	if err != nil {
-		logp.Err("Error getting process details. pid=%d: %v", process.Pid, err)
+		logp.Debug("Error getting details for process %s with pid=%s: %v", process.Name, process.Pid, err)
```

Comment:
`logp.Debug` will need a selector as first parameter, I'm also curious, I guess you changed this because it would be too noisy for non privileged users?

Reply:
@exekias, done, somehow, I missed it previously. As briefly mentioned in the description, the initial change (to report all processes , but information about which users owns those processes will be missing https://github.com/elastic/beats/pull/12305) will result in a access denied errors in process.getDetails. This will break some of the tests in the CI as they did not expect any error type messages and indeed it will be noisy for the non privileged users. I have made sure the behavior is consistent in getsingleprocess function and return debug messages at all steps.